tox.ini

```diff
@@ -3,12 +3,13 @@ envlist=lint, py{36,37,38}
 skipsdist=True
 
 [pytest]
-addopts=-m 'standard and not integration and not long_running'
+addopts=-m 'standard and not integration and not long_running and not public_api'
```

Comment:
Why do we not want to test this every time? I don't think it's that time consuming, or is it?